src/x11/xf86WacomProperties.c

```diff
@@ -931,17 +931,22 @@ static int wcmSetProperty(DeviceIntPtr dev, Atom property, XIPropertyValuePtr pr
 			common->wcmPressureRecalibration = values[0];
 	} else if (property == prop_panscroll_threshold)
 	{
-		CARD32 *values = (CARD32*)prop->data;
+		INT32 *values = (INT32*)prop->data;
```

Comment:
huh, interesting, it was already an `INT32` anyway, so this isn't even an API break.